packages/ui-components/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@cfxjs/ui-components",
+  "type": "module",
+  "main": "components/index.js",
```

Comment:
add version as 0.0.0 to this file, make a private if we don't want to publish it